components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/util/ClaimMetaDataCache.java

```diff
@@ -28,10 +32,15 @@
     private static final String CLAIM_META_DATA_CACHE_NAME = "ClaimMetaDataCache";
 
     private static ClaimMetaDataCache instance = new ClaimMetaDataCache();
+    private static final List<AbstractCacheListener<ClaimMetaDataCacheKey, ClaimMetaDataCacheEntry>>
```

Comment:
Do we need these listeners now?
Remember they were used when we had distributed caches. If they are not used can we get rid of them now.